contracts/Journal.sol

```diff
@@ -15,11 +16,12 @@ contract Journal is Ownable {
     /*
         This function should be called by the author of the paper.
     */
-    function addPaper(string memory _ipfsHash, address _paper) public {
-        allPapers[_ipfsHash] = Paper(_paper);
-        if (owner() != allPapers[_ipfsHash].owner())
-            allPapers[_ipfsHash].transferOwnership(owner());
-        allPapers[_ipfsHash].addPublisher(owner());
+    function addPaper(string memory _ipfsHash, uint _maxReviewTime) public returns (Paper) {
+        Paper paper = new Paper(_ipfsHash, new string[](0), msg.sender, _maxReviewTime);
+        allPapers[_ipfsHash] = paper;
+        paper.addPublisher(msg.sender);
+        emit NewPaper(paper);
+        return paper;
     }
```

Comment:
I disagree with that if it is the only change.
As you can see, the Paper owner contract can perform operations like validate reviews, which is the editor's role., and add reviewer. Therefore, when the author of the paper wants to be under an editor, it stays the author but the ownership is transfers, so that the editor can have a little more power on the paper review.